dev/integration_tests/flutter_gallery/lib/demo/pesto_demo.dart

```diff
@@ -224,7 +224,6 @@ class _PestoLogoState extends State<PestoLogo> {
         child: SizedBox(
           width: kLogoWidth,
           child: Stack(
-            overflow: Overflow.visible,
```

Comment:
now that stack defaults to clipping, you'll have to add something here to not clip, no?